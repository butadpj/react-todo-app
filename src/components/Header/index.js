import React from 'react';

const Header = (props) => <header>{props.children}</header>

export default Header
