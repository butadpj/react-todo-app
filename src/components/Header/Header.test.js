import Header from "./index";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";


describe("<Header />", () => {
  it("renders without crashing", () => {
    shallow(<Header />);
  });

  it("UI doesn't change unexpectedly", () => {
    const wrapper = mount(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('renders children when passed in', () => {
    const wrapper = shallow((<Header>Todo App</Header>));
    expect(wrapper.contains('Todo App')).toBe(true);
  });
});