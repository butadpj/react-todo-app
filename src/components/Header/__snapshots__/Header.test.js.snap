// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> UI doesn't change unexpectedly 1`] = `
<Header>
  <header />
</Header>
`;
